docs/_guide/07-tools.md

```diff
@@ -0,0 +1,194 @@
+
+* [https://github.com/43081j/eslint-plugin-lit](https://github.com/43081j/eslint-plugin-lit)
+
+### Dev server
+
+
+lit-html is packaged as JavaScript modules. Many developers prefer to import modules using node-style module identifiers, which aren't supported yet by browsers. To run in the browser, these module identifiers need to be transformed to browser-ready module identifiers. The Polymer dev server, which is part of the Polymer CLI, performs this transformation on the fly, so you can preview your project during development without a build step. Another alternative is the Open Web Components Dev Server (`owc-dev-server`).
+
+If you're using webpack for your build process, you can also use the webpack dev server.
+
+#### Polymer Dev Server
+
+Install the Polymer CLI:
+
+```bash
+npm i -g polymer-cli
+```
+
+Run the dev server:
+
+```bash
+polymer serve
+```
+
+The Polymer CLI was designed to help develop, test, and build projects using web components, JavaScript modules and other features of the modern web platform. It's not required for lit-html development, but it provides several handy utilities. 
+
+#### Open Web Components Dev Server
+
+The Open Web Components project produces a dev server that handles remapping node-style modules to browser-style modules.
+
+For full installation and usage instructions, see the [open-wc website](https://open-wc.org/developing/owc-dev-server.html). 
+
+## Testing
+
+lit-html doesn't have many special testing requirements. If you already have a testing setup, it should work fine as long as it supports working with JavaScript modules (and node-style module specifiers, if you use them).
+
+Web Component Tester (WCT) is an end-to-end testing environment that supports node-style module specifiers. works with the Mocha testing framework and (optionally) the Chai assertion library. There are two ways to add WCT to your project:
+
+* [web-component-tester](https://www.npmjs.com/package/web-component-tester).  Installing the full WCT package gives you Mocha and Chai, as well as some other add-ons.
+* [wct-mocha](https://www.npmjs.com/package/wct-mocha). Just the WCT client-side library. You'll need to install your own version of Mocha, and any other add-ons you want.
+
+Alternately, you can also use the Karma test runner. The Open Web Components recommendations includes a [Karma setup](https://open-wc.org/testing/testing-karma.html#browser-testing) that resolves module dependencies by bundling with webpack before running tests. 
+
+## Build
+
+Build tools take your code and make it production-ready. Among the things you may need build tools to do:
+
+* Transform ES6 code to ES5 for legacy browsers, including transforming JavaScript modules into other formats.
+* Bundle modules together can improve performance by reducing the number of files that need to be transferred. 
+* Minify JavaScript, HTML, and CSS.
+
+Many build tools can do this for you. Currently we recommend the Polymer CLI or webpack. 
```

Comment:
Again I'd personally recommend Rollup over WebPack, but I do see the issue in that open-wc.org doesn't have a default Rollup config. I'm going to file an issue over there.